### DISCUSSION TITLE
求助：V3版本，挂载夸克云盘显示“work”后，盘内无文件/文件夹。

### DISCUSSION THREAD

**Discussion by DN-NateRiver** [AlistGo/alist#3197] [Q&A]
alist版本：V3
夸克网盘挂载信息：
cookie：按照alist文档提取填写；
根文件夹ID：0；
webdav代理：本地代理；
填写后显示挂载状态：work，但实际打开后盘内文件/文件夹均不显示。
求各位大佬指导怎么成功挂载？谢谢

**Comment by clsddy** [comment]
可能因为你挂梯子了
